expander.c

```diff
@@ -196,11 +197,13 @@ bgpq_expander_add_as(struct bgpq_expander *b, char *as)
 	}
 
 	if (!expand_special_asn &&
-	    ((asno  >= 4200000000ul) || (asno >= 64496 && asno <= 65551)))
+	    ((asno  >= 4200000000ul) || (asno >= 64496 && asno <= 65551))) {
+		sx_report(SX_ERROR,"Invalid AS number: %u\n", asno);
 		return 0;
+	}
 
 	if ((asne = malloc(sizeof(struct asn_entry))) == NULL)
-		err(1, NULL);
+		sx_report(SX_FATAL, "malloc failed for asn\n");
```

Comment:
if `malloc()` fails we are dead anyway, calling `err(1, NULL)` is fine

Reply:
It's about consistency/code style, here FATAL is used: https://github.com/bgp/bgpq4/blob/016da02b9ebb2f103738109caf9f406374efd914/expander.c#L206

and here: https://github.com/bgp/bgpq4/blob/016da02b9ebb2f103738109caf9f406374efd914/expander.c#L541

Reply:
I prefer `err()`

Reply:
Resolved through with the exception that an int existed call `err` which is of course bad practice, I renamed it to `e`.